packages/gotrue/lib/src/gotrue_client.dart

```diff
@@ -813,8 +854,11 @@ class GoTrueClient {
         await admin.signOut(accessToken, scope: scope);
       } on AuthException catch (error) {
         // ignore 401s since an invalid or expired JWT should sign out the current session
+        // ignore 403s since user might not exist anymore
         // ignore 404s since user might not exist anymore
-        if (error.statusCode != '401' && error.statusCode != '404') {
+        if (error.statusCode != '401' &&
+            error.statusCode != '403' &&
```

Comment:
This edit has been made, because for deleted users on the latest Supabase auth server, the following error is returned:
`AuthException(message: User from sub claim in JWT does not exist, statusCode: 403)`